src/ws/Makefile-ws.am

```diff
@@ -77,7 +77,7 @@ noinst_DATA += cockpit-testing.service
 testassets_systemdunit_data += cockpit-testing.service
 
 cockpitconfdir = $(sysconfdir)/cockpit
-cockpitconf_DATA = src/ws/config
+cockpitconf_DATA = src/ws/cockpit.conf
```

Comment:
Should i leave it installing config then like it was before, or remove it all together?